tests/test_telegram.py

```
@@ -146,46 +161,20 @@ def test_correct_password_inserts_user(self, mock_send):
-            "from": {"id": 12345},
-            "chat": {"id": 12345},
-            "text": "/start wrong-password",
-        }
-        handle_message(msg, mock_conn, cfg)
-        mock_send.assert_called_once()
-        assert "Invalid token" in mock_send.call_args[0][1]
+        assert "Welcome" in mock_send.call_args[0][1]
 
     @patch("api.telegram.send_message")
     def test_already_authorized_user_gets_message(self, mock_send):
         mock_conn, mock_cursor = _mock_cursor_returning({"user_id": 12345})  # already allowed
```

Ruff (0.15.12)

[warning] Unpacked variable `mock_cursor` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)